docs/changelog/0004-add-analytics-queue.md

```diff
@@ -0,0 +1,20 @@
+# 4. Add queue for analytics tasks
+
+**Date:**  2018-08-16
+
+**Optional per env:** Yes
+
+**Dependant CommCare version:** None
```

Comment:
can you update this with the merge commit on the PR which requires it: https://github.com/dimagi/commcare-hq/commit/a507757628bc5c087fd1badc0145e39c5bf790ae